src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -84,31 +80,6 @@ public void parse_optionalFieldsMissing_success() {
                         + CLASS_GROUP_DESC_AMY + STUDENTID_DESC_AMY, new AddCommand(expectedStudent));
     }
 
-    @Test
-    public void parse_compulsoryFieldMissing_failure() {
```

Comment:
Is there a test case for the test case for missing compulsory fields of "Name" and "StudentId"

Reply:
Nope, I'll find time to add them. Thanks.